soql-stdlib/src/main/scala/com/socrata/soql/functions/Docs.scala

```diff
@@ -0,0 +1,88 @@
+package com.socrata.soql.functions
+
+import com.socrata.soql.types._
+
+object Docs {
+    def main(args: Array[String]) {
+        val out_path = args(0)
+        SoQLFunctions.allFunctions.filter(x => x.name.toString != "op$[]").foreach { function => 
```

Comment:
Generally preferable to use the `SpecialFunctions.Operator` matcher rather than hardcoding `op$` anywhere (same below, in CleanName; all `SpecialFunctions` all the time)